apps/backend/src/iot-devices/application/use-cases/rotate-certificate/rotate-certificate.ts

```
@@ -0,0 +1,72 @@
+    const newCertResult = await this.awsIot.createKeysAndCertificate();
+    if (newCertResult.isFailure()) {
+      await this.deviceRepository.updateStatus(thingName, "active");
+      return newCertResult;
+    }
+
+    const { certificateId, certificateArn } = newCertResult.value;
+
+    const attachResult = await this.awsIot.attachThingPrincipal(thingName, certificateArn);
+    if (attachResult.isFailure()) {
+      const r1 = await this.awsIot.updateCertificateStatus(certificateId, "REVOKED");
+      if (r1.isFailure()) this.logger.warn({ msg: "Cleanup failed: revoke new cert", certificateId, error: r1.error });
```

ESLint

[error] Replace `·this.logger.warn({·msg:·"Cleanup·failed:·revoke·new·cert",·certificateId,·error:·r1.error` with `⏎········this.logger.warn({⏎··········msg:·"Cleanup·failed:·revoke·new·cert",⏎··········certificateId,⏎··········error:·r1.error,⏎·······`

(prettier/prettier)